lms/static/js/student_account/views/AccessView.js

```diff
@@ -163,7 +163,8 @@
                         model: model,
                         thirdPartyAuth: this.thirdPartyAuth,
                         platformName: this.platformName,
-                        ipAddressWarningMsg: this.registerIpAddressWarningMsg // Added by Labster
+                        ipAddressWarningMsg: this.registerIpAddressWarningMsg, // Added by Labster
+                        accountCreationWarningMsg: this.accountCreationWarningMsg // Added by Labster
```

Comment:
`// Added by Labster` was added here, but the line above not. What's the case? :)

Reply:
added